maintainer/docker/opensuse/Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM opensuse:42.1
+FROM opensuse:42.3
 MAINTAINER Florian Weik <fweik@icp.uni-stuttgart.de>
 RUN zypper -n --gpg-auto-import-keys refresh \
 && zypper -n --gpg-auto-import-keys install \
-  patterns-openSUSE-devel_C_C++ \
+  patterns-openSUSE-devel_C_C++ gcc-c++ \
   cmake \
-  openmpi-devel \
+  openmpi-devel Modules \
   fftw3-devel \
   boost-devel libboost_serialization1_54_0 libboost_mpi1_54_0  libboost_filesystem1_54_0 libboost_test1_54_0 \
-  python python-numpy python-PyVTK python-h5py\
-  tcl-devel \
```

Comment:
Can also be solved by a repository for the docker files. Than we can branches mirroring those of the code repro that go to different container tags.